### ISSUE TITLE
Fix duplication of in-page ToC

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5296]
a11y script found duplicate ID for the in-page ToC. The ToC is rendered twice: once for desktop layout and once for mobile layout. This raises a [WCAG F77 error](https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/#sc4_1_1) on pages with in-page ToCs.  This causes issues with accessibility technology. This should be fixed.
